java/spring/container/Scripts/promote-groups.sh

```diff
@@ -1,16 +1,20 @@
 #!/bin/sh
+# Delete old deprecated resource groups before setting current prod to deprecated.
 ./delete-groups-by-tag.sh Environment=deprecated
 echo "Querying for production resource groups."
 groups=$(az group list --tag Environment=production --query "[].name" --output tsv)
 for name in $groups
 do
-	echo "Updating tags for resource group:" $name
-	az group update --name $name --tags Environment=deprecated
+	./update-group-tag-by-name.sh $name Environment=deprecated
 done
-echo "Querying for pre production resource groups."
+echo "Querying for pre-production resource groups."
 groups=$(az group list --tag Environment=pre-production --query "[].name" --output tsv)
 for name in $groups
 do
-	echo "Updating tags for resource group:" $name
-	az group update --name $name --tags Environment=production
-done
\ No newline at end of file
+	./update-group-tag-by-name.sh $name Environment=production
+done
+# Output production group list to file.
+mkdir -p logs/
+destdir=logs/promotion-list
+echo "Saving list of updated resource groups to logs/promotion-list"
+echo "$groups" > "$destdir"
```

Comment:
Are you wanting to keep appending to logs/promotion-list or to overwrite it? You probably want ```>>``` here.